### ISSUE TITLE
Kraken Good Accessibility - Completed

### ISSUE THREAD

**Issue by mmcgreg** [mmcgreg/kraken-good-accessibility#1]
@thatkidmadrid

**Comment by thatkidmadrid** [comment]
## TL;DR;

Great! 🎉 
Nothing really to revisit and fix for the grade, but I have some feedback to help bring it to the next level
- The image alt attribute description is very informative! But I have a trick that is very small, but goes a long way to add a valuable layer of description.
- Although the aria-labels you used on the icons was good, there was a elegant solution to the aria-labels hinted by the icon names.

## Feedback
The structure demonstrates that you understand what the basic html tags are used for. This makes navigating using screen readers make more sense and easier 👍 

Although your description is already very descriptive, I want to share a little trick that adds a lot of value to an image desscription. You had `A kraken wrapping itself around a ship filled with sailors. The ship is sinking.`, but I think this could be made even better with `a black and white illustration of A kraken wrapping itself around a ship filled with sailors. The ship is sinking.` just something small like that on top of what you already had provides a lot more context. There's no ambiguity of if it's a photo, a 3d-render, etc. This small addition does a good job of clarifying.

That being said for this application I don't need to be overly detailed like `a black and white charcoal illustration of the 8-armed sea monster, the kraken, with its arms surrounding a half-submerged ship with a ship and seagulls in the background`

Use your judgement, put yourself in the listener's shoes, and think of a description that would help _you_ understand the image. In some applications (like a design portfolio 😉 ) a more detailed description may be useful, but then that introduces other accessibility options like `aria-details`

You've demonstrated that you understand that the aria-labels that use roman numerals can be confusing, so  seeing that you opted to use numbers instead is great!

> **me being nitpicky**: `xvii` is `17` and `xix` is `19` instead of 57 and 59

I think there was an even more elegant solution - that was hinted by the name of the icons - by just using `previous` and `next` instead of the number of the chapters. I bring this up because a dynamic header that will swap out the previous and next chapter numbers for each chapter's page would require some function that can recognize the current chapter and to calculate and insert into the aria-labels - or we would have to manually create new headers for every chapter. This isn't an accessibility concern, but later on you will be introduced to the idea of reusable code components

**Comment by thatkidmadrid** [comment]
check changed from yellow to green on LTW 👍
